vertical-pod-autoscaler/deploy/vpa-v1-crd-gen.yaml

```diff
@@ -383,6 +383,12 @@ spec:
                   pods. If not specified, all fields in the `PodUpdatePolicy` are
                   set to their default values.
                 properties:
+                  minReplicas:
+                    description: Overrides global '--min-replicas' flag, specifying
+                      minimal number of replicas which need to be alive for Updater
+                      to attempt pod eviction (pending other checks like PDB).
```

Comment:
Maybe we should add a validation to ensure that this is a positive value?  (I think 0 and negative values don't make sense).

If we do then we should document that

Reply:
Good idea. I coupled CRD changes and validation into a separate commit.